Storage/src/Connection/RetryTrait.php

```diff
@@ -28,6 +28,7 @@ trait RetryTrait
      */
     private $httpRetryCodes = [
         0, // connetion-refused OR connection-reset gives status code of 0
+        200, // partial download cases
```

Comment:
This is what we received in partially downloaded files. Does this seem correct? I thought it should have been 204.

Reply:
So, we do receive an exception in cases like: `return-broken-stream-after-8096K` but the status code is still 200(as far as I observed), even I expected "partial content".

Reply:
Ah, it would be good to clarify what case this is intended for then. It would also be good to clarify how the exception is caught and handled.

Reply:
Resolved offline :)